cmd/neofs-cli/modules/control/shards_set_mode.go

```diff
@@ -17,9 +17,10 @@ const (
 	shardIDFlag          = "id"
 	shardClearErrorsFlag = "clear-errors"
 
-	shardModeReadOnly  = "read-only"
-	shardModeReadWrite = "read-write"
-	shardModeDegraded  = "degraded"
+	shardModeReadOnly         = "read-only"
+	shardModeReadWrite        = "read-write"
+	shardModeDegraded         = "degraded"
```

Comment:
Let's rename this state to `degraded-read-write` for the sake of consistency.

Reply:
@fyrchik

Reply:
Fixed